src/app/page/Error/Unauthorized.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { Link as RouterLink } from "react-router-dom";
+import clsx from "clsx";
+import makeStyles from "@material-ui/core/styles/makeStyles";
+
+import { Button } from "../../component";
+import { Body1, H1 } from "../../component/Typography";
+import unauthorizedImg from "../../../img/app-illustrations/error_401.png";
+import { routes } from "../../routing";
+
+const unauthorizedStyles = makeStyles((theme) => ({
+  graphic: {
+    height: "100%",
+    width: "100%",
+  },
+  heading: Object.assign(theme.typography.h1, {
```

Comment:
this vs spread operator? just curious about the choice

Reply:
lol its old school way, since we can use ES6, I'll change it to spread